src/data/markdown/docs/30 xk6-browser/02 API/03 BrowserType/launch--options--.md

```diff
@@ -13,12 +13,58 @@ Launches a new browser process.
 | env               | string[] | `null`  | Environment variables to set before launching browser process.                                                                                                                                                                                                        |
 | executablePath    | string   | `null`  | Override search for browser executable in favor of specified absolute path.                                                                                                                                                                                           |
 | headless          | boolean  | `true`  | Show browser GUI or not.                                                                                                                                                                                                                                              |
-| ignoreDefaultArgs | string[] | `null`  | Ignore any of the default arguments included when launching browser process.                                                                                                                                                                                          |
+| ignoreDefaultArgs | string[] | `null`  | Ignore any of the [default arguments](#default-arguments) included when launching browser process.                                                                                                                                                                                          |
 | proxy             | string   | `null`  | Specify to set browser's proxy configuration.                                                                                                                                                                                                                         |
 | slowMo            | string   | `null`  | Slow down input actions and navigation by the specified time e.g. `'500ms'`.                                                                                                                                                                                          |
 | timeout           | string   | `'30s'` | Default timeout to use for various actions and navigation.                                                                                                                                                                                                            |
 
 
+### Default arguments
+
+List of default arguments included when launching the browser process.
+
+**Note:** The starting '--' have been omitted from the argument names in these lists.
```

Comment:
Should we use a "note box" as in [here](https://mdr-ci.staging.k6.io/docs/refs/pull/1042/merge/javascript-api/xk6-browser/get-started/selecting-elements/)?

Reply:
Done in https://github.com/grafana/k6-docs/pull/1042/commits/960625a7972c3be6363079c49438b38ed6f0e7b9 plus [fixup](https://github.com/grafana/k6-docs/pull/1042/commits/1c18330684d8fad9bb82f075ba3e7e1e49f0aac5).